upgrade/upgrade-scripts/verify-jar

```diff
@@ -55,48 +50,147 @@ function verify_jar_verify_cleanup() {
 	return "$rc"
 }
 
+#
+# Use formatted usage
+#
+function usage() {
+	echo "$(basename "$0"): $*" >&2
+
+	PREFIX_STRING="Usage: $(basename "$0")"
+	PREFIX_NCHARS=$(echo -n "$PREFIX_STRING" | wc -c)
+	PREFIX_SPACES=$(printf "%.s " $(seq "$PREFIX_NCHARS"))
+
+	echo "$PREFIX_STRING [-d] [-f format] [-l locale] [-o path]"
+	echo "$PREFIX_SPACES [-fsCheckStartTs ts] [-fsCheckEndTs ts]"
+	echo "$PREFIX_SPACES [-containerInitStartTs ts] [-containerInitEndTs ts]"
+
+	exit 2
+}
+
+OPTIONS_LONG=(
+	"containerInitStartTs:"
+	"containerInitEndTs:"
+	"fsCheckStartTs:"
+	"fsCheckEndTs:"
+	"help"
+)
+
+OPTIONS_SHORT=(
+	"d"
+	"f:"
+	"h"
+	"l:"
+	"o:"
+)
+
+GETOPT=$(getopt --alternative \
+	--name "$(basename "$0")" \
+	--options "$(printf "%s," "${OPTIONS_SHORT[@]}")" \
+	--longoptions "$(printf "%s," "${OPTIONS_LONG[@]}")" \
+	-- "$@") || usage "$@"
+eval set -- "$GETOPT"
+
+#
+# Add support for optional timestamp parameters from managment stack. These timestamps
```

Comment:
IMO, this comment isn't very useful.. but if you think it is, perhaps we can at least move it to be closer to the options it's commenting on? e.g. move it directly above the `--container-InitStartTs`?